cldf/forms.csv

```diff
@@ -138,7 +138,7 @@ kkhalqabad1-tolaugh-1,,kkhalqabad1,tolaugh,kʉlʊː,kʉlʊː,k ʉ l ʊː,,Mennec
 kkhalqabad1-tolie-1,,kkhalqabad1,tolie,ʒatʊː,ʒatʊː,ʒ a t ʊː,,Mennecier2016,,
 kkhalqabad1-tolive-1,,kkhalqabad1,tolive,ʒasaʊ,ʒasaʊ,ʒ a s a ʊ,,Mennecier2016,,
 kkhalqabad1-toplay-1,,kkhalqabad1,toplay,wɔjnaʊ,wɔjnaʊ,w ɔ j n a ʊ,,Mennecier2016,,
-kkhalqabad1-topull-1,,kkhalqabad1,topull,itʲerʊ,itʲerʊ,i tʲ e r ʊ,,Mennecier2016,,
+kkhalqabad1-topull-1,,kkhalqabad1,topull,sɔzʊ,sɔzʊ,s ɔ z ʊ,,Mennecier2016,,
```

Comment:
Important: this is a change of the entire form for the same language, right? Can you please check what is the reason for this change, @tresoldi ?

Reply:
Wow, @LinguList , that was a good catch!

So, there are two columns glossed as `to pull (тянуть)` in the [raw data](https://github.com/lexibank/cals/blob/master/raw/Table_S2_Supplementary_Mennecier_et_al..docx): the first starting on p.46, the second on p. 54 . Both `itʲerʊ` and `sɔzʊ` would be right, the first in the `raw/12.csv` file, the second in the `raw/14.csv` one. The difference is due to reproducibility: as `self.raw_dir.glob(".csv")` returns an unsorted list, different words will be the selected synonym each time we run the script. The fix is obvious, just sort the list.

However, it seemed strange to have these repeated glosses, so I went to check with Kazakh and Uzbek dictionaries. Sure enough, the `itʲerʊ` set has a wrong gloss in the supplementary material, because it actually means "to push" (a concept otherwise missing). Even Google Translate has it, see [итеру](https://translate.google.com/?hl=en#view=home&op=translate&sl=kk&tl=en&text=%D0%B8%D1%82%D0%B5%D1%80%D1%83).

What should we do?

Reply:
I'd say: first compare with the doc-file, what they have there, as the raw.csv is derived from this, and then make a decision that can be pragmatic. If you think a gloss is wrong, I trust you on this, and you could ignore it, or replace it, etc.

Reply:
Yes, it is wrong in the doc-file, I also checked on Brill's website.

I will correct the gloss and sort the list returned by `glob` nonetheless.